### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by Olivier-Kango**
# Vet clinic database: query multiple tables

**Hi there 👋,**
In this project, I : 
- Create a table named `owners` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `full_name`: string
  - `age`: integer

- Create a table named `species` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string

- Modify `animals` table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  - Remove column `species`
  - Add column `species_id` which is a foreign key referencing `species` table
  - Add column `owner_id` which is a foreign key referencing the `owners` table

- All this goes in `schema.sql` file.
- Insert the following data into the `owners` table:
  - Sam Smith 34 years old.
  - Jennifer Orwell 19 years old.
  - Bob 45 years old.
  - Melody Pond 77 years old.
  - Dean Winchester 14 years old.
  - Jodie Whittaker 38 years old.

- Insert the following data into the `species` table:
  - Pokemon
  - Digimon

- Modify your inserted animals so it includes the `species_id` value:
  - If the name ends in "mon" it will be Digimon

  - All other animals are Pokemon

- Modify your inserted animals to include owner information (`owner_id`):
  - Sam Smith owns Agumon.
 aa

  - Jennifer Orwell owns Gabumon and Pikachu.
 bb

  - Bob owns Devimon and Plantmon.
 ccc

  - Melody Pond owns Charmander, Squirtle, and Blossom.
 dd

  - Dean Winchester owns Angemon and Boarmon.
ee

- These insertions and modifications happen in `data.sql` file.
----
- Write queries (using `JOIN`) to answer the following questions:
  - What animals belong to Melody Pond?
ii
  - List of all animals that are pokemon (their type is Pokemon).
jj

  - List all owners and their animals, remember to include those that don't own any animal.
kk

  - How many animals are there per species?
 ll

  - List all Digimon owned by Jennifer Orwell.
 mm

  - List all animals owned by Dean Winchester that haven't tried to escape.
nn

  - Who owns the most animals?
 OO

- All these codes are written in the `queries.sql` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,4 +125,4 @@ Feel free to check the [issues page](https://github.com/Olivier-Kango/vet_clinic
 This project is [MIT](./LICENSE.txt) licensed.
```
Comment (tresorsawasawa) [inline README.md:127]: - [ ] `schema.sql file`: As required for the project, kindly make sure you add the schema.sql file code lines of:
  - A new table with the name `owners` with their expected columns and the correct type for each one as in the **[project's requirements.](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)**
  - Another table with the name `species` with their expected columns and the correct type on each column
  - You add the code you used to modify the `animals` table on how:
    - Remove column `species`
    - Add column `species_id` which is a foreign key referencing the `species` table
    - Add column `owner_id` which is a foreign key referencing the `owners` table

Comment (tresorsawasawa) [inline README.md:127]: - [ ] `data.sql file`: Kindly make sure  you add all the code lines in the `data.sql` file you used while adding data you were required in the **[project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)**

Screenshot from 2022-12-15 20-28-20

Comment (tresorsawasawa) [inline README.md:127]: - [ ] `queries.sql` file: Kindly make sure all queries that answer the below questions are added to the `queries.sql` file and all of them should be correct. Right now, you only add screenshots and missed their code for the different specific files.
  
Screenshot from 2022-12-15 20-30-23

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Olivier-Kango 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Olivier-Kango** [comment]
Ooh 🤕  I did everything locally but forgot to push. I apologize. Let me push. Thanks @tresorsawasawa  for the correction

**Review by Deepakdanger — APPROVED** [review]
Hi @Olivier-Kango
### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**